File: 1sem/Python/Lista2/Satoshi/Ex02.py
vals:int = [0]*100
mai:int
men:int
med:float = 0.0

for i in range(100):
    vals[i] = int(input(f'Digite o termo {i+1}: '))

    if (i==0):
        mai = vals[i]
        men = vals[i]
    if (vals[i] > mai):
        mai = vals[i]
    if (vals[i] < men):
        men = vals[i]

    med += vals[i]

med = med / len(vals)
print(f'Maior: {mai}\nMenor: {men}\nMédia: {med}')

File: 2sem/Java/Lista 2/prj_java10-09-2026/src/Ex02.java
import javax.swing.JOptionPane;
public class Ex02 {
    public static void main (String args[]){
        int v[] = new int[100];
        int men=0, mai=0, m=0;
        
        for (int i=0;i<100;i++){
            v[i] = Integer.parseInt(JOptionPane.showInputDialog("Digite o termo "+(i+1)+": "));
            
            if (i == 0){
                men=v[i];
                mai=v[i];
            } else {
                
                if (v[i] < men){
                    men = v[i];
                }
                if (v[i] > mai){
                    mai = v[i];
                }
            }
            
            m += v[i];
        }
        
        System.out.println("Média dos valores: "+m/100);
        System.out.println("Maior: "+mai+"\nMenor: "+men);
    }
}
